src/components/shared/formItems/ProjectReserved.tsx

```diff
@@ -44,10 +52,24 @@ export default function ProjectReserved({
     </FormItemWarningText>
   )
 
+  // Reserved tokens received by project per ETH
+  const initialReservedTokensPerEth = (value ?? 1) * 10 ** 4
```

Comment:
**question** can we use one of the math utils for this?

perhaps `reservedRateFrom`?

10**4 is a [magic number](https://en.wikipedia.org/wiki/Magic_number_(programming)) which we should avoid

Reply:
Cleared it up a bit but figuring out how to incorporate `reservedRateFrom` is unnecessary I think